src/Tizen.Wearable.CircularUI.Forms.Renderer/IRotaryInteractionController.cs

```diff
@@ -0,0 +1,26 @@
+﻿/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Tizen.Wearable.CircularUI.Forms.Renderer
+{
+    public interface IRotaryInteractionController
```

Comment:
We used a combination of the words below mainly in the Xamarin.Forms and CircualrUI.
For example, when starting with `Bezel`, we used it with `Interaction`. And, when starting with `Rotary`, we used it with `Focus`, `Focusable` ,... .

So, I think  `IRoataryFocusObjectController`, `IRotaryFocusableController` or even `IBezelInteractionController` would be better for the sake of consistency.

Reply:
I found one exceptional case in  XF.ViewRenderer. 
```protected virtual void UpdateRotaryInteraction()```.
:-(